jwst/lib/file_utils.py

```diff
@@ -0,0 +1,20 @@
+"""File utility functions"""
+from contextlib import contextmanager
+import os
+
+
+@contextmanager
+def pushdir(directory):
```

Comment:
This looks like you've re-implemented `pathlib.Path` context manager?  Perhaps it would be too early for us to start using `pathlib.Path` objects in this library, though I have added it as an issue.  #2892.

Reply:
Yeah, feel free to ignore it for now.  I feel like getting into `pathlib` stuff in our code is a larger problem, so we can address it in some future PR.